### PR TITLE
HELP-26611: don't send the command without UUID

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/ecallmgr/src/ecallmgr_fs_command.erl [hunk 1]
```diff
@@ -26,8 +26,10 @@
 %% set channel and call variables in FreeSWITCH
 %% @end
 %%--------------------------------------------------------------------
--spec set(atom(), ne_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
+-spec set(atom(), api_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
```
Comment (fenollp) [inline applications/ecallmgr/src/ecallmgr_fs_command.erl]: `api_ne_binary()`

**Comment by jamesaimonetti** [comment]
@fenollp updated

File: applications/ecallmgr/src/ecallmgr_fs_command.erl [hunk 2]
```diff
@@ -26,8 +26,10 @@
 %% set channel and call variables in FreeSWITCH
 %% @end
 %%--------------------------------------------------------------------
--spec set(atom(), ne_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
+-spec set(atom(), api_ne_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
 set(_, _, []) -> 'ok';
+set(_Node, 'undefined', _Props) ->
+    lager:debug("no UUID for setting on node ~s: ~p", [_Node, _Props]);
```
Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_command.erl]: @jamesaimonetti can we raise the level to `warning` ? if this happens there are problems elsewhere that need to be verified

Comment (jamesaimonetti) [inline applications/ecallmgr/src/ecallmgr_fs_command.erl]: @lazedo done

**Pull request merged**